sao.js

```diff
@@ -81,6 +81,12 @@ module.exports = {
       message: 'Author name',
       default: ':gitUser:',
       store: true
+    },
+    pm: {
+      message: 'Choose a package manager',
+      choices: ['npm5', 'yarn'],
+      type: 'list',
+      default: 'npm5'
```

Comment:
don't think it's a good idea to put the `npm` version here (will be a PITA to update this after each major release of npm)